ckan/templates/macros/form/resource_validation_errors.html

```diff
@@ -17,21 +17,25 @@
 
 
 {# (canada fork only): error -> danger BS version #}
-{% macro resource_validation_errors(errors={}, type="danger", classes=[], pkg_name='', helpers={}) %}
+{% macro resource_validation_errors(errors={}, type="danger", classes=[], pkg_name='', dataset_type='dataset', helpers={}) %}
   {% if errors %}
     <div class="error-explanation alert alert-{{ type }}{{ " " ~ classes | join(' ') }}">
-      <h3>{{ h.humanize_entity_type('package', dataset_type, 'other errors') or _('Errors in dataset') }}</h3>
+      <h3>{{ helpers.humanize_entity_type('package', dataset_type, 'other errors') or _('Errors in dataset') }}</h3>
       {% if 'dataset' in errors and errors.dataset %}
         {% set dataset_link = helpers.url_for('dataset.read', id=pkg_name) %}
-        <p>{{ h.humanize_entity_type('package', dataset_type, 'other errors package').format(dataset_link) or _('The <a href="{}" target="_blank">dataset</a> contains errors:').format(dataset_link) }}</p>
+        {% set humanized_label = helpers.humanize_entity_type('package', dataset_type, 'other errors package') %}
+        {% if humanized_label %}
+          {% set humanized_label = humanized_label.format(dataset_link)|safe %}
+        {% endif %}
+        <p>{{ humanized_label or _('The <a href="{}" target="_blank">dataset</a> contains errors:').format(dataset_link) }}</p>
```

Comment:
Do you mean
```suggestion
        <p>{{ (humanized_label or _('The <a href="{}" target="_blank">dataset</a> contains errors:')).format(dataset_link) }}</p>
```
Still, the `|safe` on line 28 won't affect the string here IIUC. Do we really need the link in the error message? Would be a lot simpler without it.